### DISCUSSION TITLE
[BUG] Payment Reward for pooling farm did not come to the correct address

### DISCUSSION THREAD

**Discussion by DimitriBalder** [Chia-Network/chia-blockchain#8197] [Support]
Currently, I'm farming in the official chia pool

A few days ago I found a block. 
The Block: 676362
Block stats on: [Chai Explorer](https://www.chiaexplorer.com/blockchain/block/0x89E8416eA5b69863CCb34D3A5C74A86bf554914724d8440bb06c58338053d9fa)

**The problem is that the reward has been rewarded to another address**

My XCH addresses:
 - Current address: `xch1n4952yna1AKVrrsso2Vu8Ke6GEY5W51wwPPMqKZJowXQCeadKTrCJ7w0ln`
 - First XCH address: `xch1kwgd0zzts5mync6d6s2fdc9hp5tj8cfch7c02wx0ytc4cm3nyh4qnsw48j`

**Screenshots**
![image](https://user-images.githubusercontent.com/24438859/129271294-ae21cc96-b67f-427a-b06f-63958c92134c.png)
![image](https://user-images.githubusercontent.com/24438859/129271332-2d344fb6-a623-4796-b894-69da248abf86.png)
![image](https://user-images.githubusercontent.com/24438859/129273101-9f46f9ef-8b9b-4db5-bf22-136a2cf5f599.png)

Proofs of found block: 
I'm farming on Pool Space
Notification from Pool:
![image](https://user-images.githubusercontent.com/24438859/129271996-2246d9d2-0205-4cb1-9cb1-823b8843f808.png)

My stats link: [here](https://pool.space/account/Balder)

On Chia explorer the reward address is: 
`xch1AKVrrsso2Vu8Ke6GEY5W51wwPPMqKZJowXQCeacsf0ptx5x26gusaey9dv`
ScreenShot: 
![image](https://user-images.githubusercontent.com/24438859/129271536-d77e6e54-0389-4036-92bb-7df818069bb9.png)

Please help me as I'm afraid for the next rewards

**Comment by loppefaaret** [comment]
Block 676362 was solved with a k32 plot with ID `9468865c2994c89882fb54b0d0070df36c1dd1f5f1dbaf1c873601bba6b88681acc8934b85925451d2995ca4709ac353`
do you own a plot file with this ID ? (the part after timestamp in the filename)

**Comment by DimitriBalder** [comment]
Hey, @loppefaaret, Thank you for your answer. I want to bring up that part after the timestamp in the filename has 64 lengths. This Plot Public key has 98 lenght. Are you sure the Plot Public key is part of the file name? Or there is another method to check if I have a plot with this plot public key?

**Comment by Gazza7zs** [comment]
@DimitriBalder , he is asking you to verify that you have (own) the plot first. 94688......PLOT, Not referring to the key yet.
That's how I interoperate @loppefaaret comment anyway.

**Comment by keliew** [comment]
FWIW, just do a `chia plots check -g 9468865c2994c8988`, and it _should_ find the plot file.

I'd double check and change the [payout_address](https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#payout-addresses) accordingly.

**Comment by DimitriBalder** [comment]
Hey @Gazza7zs, @keliew, thank you for your help. I was trying to find this plot with the strat name `94688`, unfortunately, I did not find such a plot.

**Comment by emlowe** [comment]
Block 676362 farmer rewards were sent to `xch17pdwsszvj3zmyap763gqpjlvyfmwwhm378w35tcsf0ptx5x26gusaey9dv`

Note, you need to check your logs to see if you really farmed the block - you should see various log entries about farming unfinished blocks.

**Comment by BrandtH22** [comment]
@DimitriBalder , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
